spec/chromium-spec.js

```diff
@@ -166,6 +166,26 @@ describe('chromium feature', function() {
       b = window.open("file://" + fixtures + "/pages/window-opener-node.html", '', 'nodeIntegration=no,show=no');
     });
 
+    it('disables node integration when it is disabled on the parent window', function(done) {
+      var b;
+      listener = function(event) {
+        assert.equal(event.data, 'undefined');
```

Comment:
It would be nice to come up with an assertion that's less abstact. I know when we paired we tested the failure case before adding the implementation, but on its own this code gives me very little confidence that the feature actually works.